src/app/App.test.tsx

```diff
@@ -67,6 +69,17 @@ describe("App", () => {
     );
   });
 
+  it("displays an error if vault is unreachable (sealed)", () => {
+    state.config.errors = "Error!!!!";
+    state.status.authenticated = true;
+    const store = mockStore(state);
+    renderWithBrowserRouter(<App />, { route: "/settings", store });
```

Comment:
You can pass state directly to `renderWithBrowserRouter` (meaning you can remove line `75, 1 line of code less).